src/components/TokenTotalLineItem.tsx

```diff
@@ -37,13 +45,23 @@ export default function TokenTotalLineItem({
         title={title ?? t('total')}
         textStyle={fontStyles.regular600}
         amount={
-          <LegacyTokenDisplay
-            amount={tokenAmount.plus(feeInToken ?? 0)}
-            tokenAddress={tokenAddress}
-            localAmount={localAmount}
-            hideSign={hideSign}
-            testID="TotalLineItem/Total"
-          />
+          tokenId ? (
```

Comment:
if you were to keep the implementation you have-- which I still think you probly shouldn't-- what's the point of these usages of LegacyTokenDisplay? Aren't you guaranteed to have a tokenId by this point from `tokenInfo`?